library/error.c

```diff
@@ -109,6 +109,10 @@
 #include "mbedtls/entropy.h"
 #endif
 
+#if defined(MBEDTLS_ERROR_C)
+#include "mbedtls/error.h"
```

Comment:
`error.h` is already being included, so we shouldn't need to include it again here.